modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc

```diff
@@ -3,14 +3,23 @@
 
 [WARNING]
 ====
+This feature was deprecated in Bonita v2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
+but it will not be offered to new customers, nor offered as a modification for existing Bonita subscription customers who are not already using multi-tenancy.
+====
+
+[NOTE]
```

Comment:
```suggestion
[IMPORTANT]
```

Reply:
I agree with the proposal as it clarifies the intention. In this case, we can remove 'Important Note' from the text
See https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/